SwiftWisdom/SwiftWisdom/Core/UIKit/UIView+GetConstraints.swift

```diff
@@ -59,27 +59,32 @@ public extension UIView {
     }
     
     /**
-    More precise way of getting constraints for a view. The constraint does not have to belong to the reciever -- this method searches the superview for the matching constraint as well.
-    
-    :param: attribute     Attribute to look for belonging to the receiver
-    :param: toItem        Probably a UIView, that the receiver is related to.
-    :param: itemAttribute Attribute that should be used with the toItem
-    
-    :returns: The first constraint that matches. May return unexpected constraint if receiver contains multiple constraints with this item and itemAttribute.
-    */
-    public func ip_constraintForAttribute(attribute: NSLayoutAttribute, onView view1: UIView? = self, toView view2: UIView? = nil, viewAttribute: NSLayoutAttribute? = nil) -> NSLayoutConstraint? {
+     More precise way of getting constraints for a view. The constraint does not have to belong to the reciever -- this method searches the superview for the matching constraint as well.
+     
+     :param: attribute     Attribute to look for belonging to the receiver
+     :param: view1        Defaults to the receiver, but could be any subview as well.
+     :param: view2        Probably a UIView, that view1 is related to.
+     :param: viewAttribute Attribute that should be used with the toItem
+     
+     :returns: The first constraint that matches. May return unexpected constraint if receiver contains multiple constraints with this item and itemAttribute.
+     */
+    public func ip_constraintForAttribute(attribute: NSLayoutAttribute, var onView view1: UIView? = nil, toView view2: UIView? = nil, viewAttribute: NSLayoutAttribute? = nil) -> NSLayoutConstraint? {
+        
+        if view1 == nil {
```

Comment:
I think instead of this, and the subsequent `!`, I'd rather see an unwrapped `let` here.  The `var` is actually going to be removed in the next Swift release from arguments, so better to change this now.  After reading the method, I also think the external parameters make for better internal names since they're more clear.

Make internal argument `onView`  as well.  You can unwrap to same name which is common for these situations

```Swift
let onView = onView ?? self
```